Buscorepuestos/buscorepuestos-web
# Feature/br 351 header

## What type of PR is this? (check all applicable)

- [x ] Styles

## Description

The redesign of the header for both desktop and mobile versions has been completed. The other components have been commented out as many of them caused overflow issues in the design. Therefore, it was decided to develop each component individually with its own styles.


## Related Tickets & Documents

none

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

Also make sure you check this browser compatibility version for desktop and mobile:
- [x ] Chrome 
- [ x] Firefox

## Checklist

DO NOT send the PR without completing the checklist above:

- [x ] Code works.
- [x ] Check all variables if they are well named and in English.
- [ x] Check responsive
- [x ] Re-check if you completed all the requirements of the task.

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] No, and this is why: They were just style changes.

## Are there any post deployment tasks we need to perform? No
